tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/query/QueryNegativeTests.java

```diff
@@ -127,4 +128,25 @@ public void testFromClauseWithInvalidAssignmentToFinalVar() {
         validateError(compileResult, index++, "cannot assign a value to final 'item'", 103, 13);
         Assert.assertEquals(compileResult.getDiagnostics().length, index);
     }
+
+    @Test
+    public void testQueryEmptyTuple() {
+        CompileResult compileResult = BCompileUtil.compile("test-src/query/query_empty_tuple.bal");
```

Comment:
Shall we rename these bal files to `<bal-file-name>_negative.bal` format? When we add negative tests we follow that format for file names.Refer to other tests which were written before